weed/s3api/s3acl/acl_helper.go

```diff
@@ -405,13 +405,23 @@ func GetAcpGrants(entryExtended map[string][]byte) []*s3.Grant {
 
 // AssembleEntryWithAcp fill entry with owner and grants
 func AssembleEntryWithAcp(objectEntry *filer_pb.Entry, objectOwner string, grants []*s3.Grant) s3err.ErrorCode {
-	objectEntry.Extended[s3_constants.ExtAmzOwnerKey] = []byte(objectOwner)
-	grantsBytes, err := json.Marshal(grants)
-	if err != nil {
-		glog.Warning("assemble acp to entry:", err)
-		return s3err.ErrInvalidRequest
+	if objectEntry.Extended == nil {
+		objectEntry.Extended = make(map[string][]byte, 0)
```

Comment:
remove this zero here, and other similar places.

Reply:
@chrislusf done